sentry-spring/src/main/java/io/sentry/spring/reactive/SentryReactiveErrorAttributes.java

```diff
@@ -0,0 +1,15 @@
+package io.sentry.spring.reactive;
+
+import com.jakewharton.nopen.annotation.Open;
+import org.springframework.boot.web.reactive.error.DefaultErrorAttributes;
+import org.springframework.web.server.ServerWebExchange;
+
+@Open
+public class SentryReactiveErrorAttributes extends DefaultErrorAttributes {
```

Comment:
`DefaultErrorAttributes` comes from Spring Boot and we cannot use it in `sentry-spring` module. 

Shouldn't we implement `WebExceptionHandler` to capture exception instead?

Reply:
@maciejwalkowiak thanks for take the time to make the review!, You're right, I was heading at the wrong direction with the DefaultErrorAttributes, I'll change it to WebExceptionHandler